packages/components/package.json

```diff
@@ -51,7 +51,9 @@
     "build.package": "pnpm build.package/create-index && vite build --config vite.config.package.js && pnpm build.package/remove-index",
     "build.package/create-index": "node scripts/node-generate-package-index.cjs && echo '✅ Index created'",
     "build.package/remove-index": "node -e \"require('fs').unlinkSync('src/solid-components.package.ts')\" && echo '✅ Index removed'",
-    "build.storybook": "storybook build --output-dir dist/storybook && cp dist/custom-elements.json dist/storybook/custom-elements.json",
+    "build.storybook": "storybook build --output-dir dist/storybook && cp dist/custom-elements.json dist/storybook/custom-elements.json && pnpm build.storybook-page-title && pnpm build.storybook-favicon",
+    "build.storybook-page-title": "node scripts/storybook/node-modify-page-title.cjs",
+    "build.storybook-favicon": "node scripts/storybook/node-modify-favicon.cjs",
```

Comment:
I would strongly suggest to just write one single command, like `modify-storybook` or similar. Who knows what will come.

As it is a subtask I furthermore suggest to write the command as `build.storybook/modify-storybook`, as this is not a build task that works on is own but always together with `build.storybook`.